### DISCUSSION TITLE
QRControl Version 1.8.0

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#117] [Announcements]
Added support for GoPro HERO11 cameras.
Added Zoom control to be combined with the Lens selection.
Separated color controls from the bitrate selection.
Protune layout organized to match the later camera models.

There is a still a yet-to-found white-screen bug that happens to about 5% of users when version updates.  I believe this to be some database corruption as it is solved by erasing the apps storage. If this happens for you: on iOS uninstall QRControl and reinstall from the App Store, on Android Settings->Apps->QRControl->Storage & Cache, then press "Clear storage" and "Clear cache".

**Comment by fxstein** [comment]
Thanks! Appreciate the update. Need to re-test some of the settings on the 11s. Before things like ISO were broken.
